jmclient/jmclient/wallet_rpc.py

```diff
@@ -1023,3 +1047,111 @@ def getseed(self, request, walletname):
                 raise InvalidRequestFormat()
             seedphrase, _ = self.services["wallet"].get_mnemonic_words()
             return make_jmwalletd_response(request, seedphrase=seedphrase)
+
+        @app.route('/wallet/<string:walletname>/taker/schedule', methods=['POST'])
+        def start_tumbler(self, request, walletname):
+            self.check_cookie(request)
+
+            if not self.services["wallet"]:
+                raise NoWalletFound()
+            if not self.wallet_name == walletname:
+                raise InvalidRequestFormat()
+
+            # -- Options parsing -----------------------------------------------
+
+            (options, args) = get_tumbler_parser().parse_args([])
+            self.tumbler_options = vars(options)
+
+            # At the moment only the destination addresses can be set.
+            # For now, all other options are hardcoded to the defaults of
+            # the tumbler.py script.
```

Comment:
I think this is a sensible simplification for a first step. Obviously in future we can `POST` a json of all the options (admittedly there are many!)